app/konnect/updates.md

```diff
@@ -11,6 +11,10 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## May 2024
 
+**Gateway 3.7 Support**
+: We've enhanced {{site.konnect_short_name}} by integrating key features from {{site.base_gateway}} version 3.7. Highlights include Plugin Streaming, Dynamic Log Leveling, and a refreshed OTEL Plugin, among other improvements!
+: The expression router is fully supported in the {{site.konnect_short_name}}. You can now define your routes via expressions, and perform CRUD operations on them.
```

Comment:
```suggestion

: The expression router is fully supported in the {{site.konnect_short_name}}. You can now define your routes via expressions, and perform CRUD operations on them.
```

Reply:
This item doesn't seem like a 3.7 change, wonder if it should be a separate entry? The expressions router has been supported in Gateway UIs in a while.